dot/network/config.go

```diff
@@ -104,6 +107,11 @@ type Config struct {
 	telemetryInterval time.Duration
 
 	noPreAllocate bool // internal option
+
+	batchSize int // internal option
+
+	// Babe slot duration
+	SlotDuration time.Duration
```

Comment:
nit shouldn't the comment begin with `SlotDuration`? Also maybe rename it to `BabeSlotDuration`?

Reply:
Done

Reply:
I have removed `Babe` from comments since it is the slot duration to produce block but the algorithm may change in future.